### ISSUE TITLE
New fast growing Delta sublineage with 4 distinct Spike mutations observed in multiple European countries

### ISSUE THREAD

**Issue by corneliusroemer** [cov-lineages/pango-designation#215]
Proposal for new sublineage within B.1.617.2
by Cornelius Roemer and @Chaoran-Chen

**Defining AA mutations**
Occurred first: S:29A, S:250I
Followed by: S:T299I
Last (defining): S:Q613H

**First observation**
2021-06-11 in Japan
Inferred common ancestor using time tree: April-June 2021

**Latest sequence**
2021-09-07

**Countries**
Observed in 25 countries, mostly in Belgium, Denmark, France, Netherlands, Germany

![image](https://user-images.githubusercontent.com/25161793/132893999-b2776daa-5f25-4df4-81a8-2fb2a858ef05.png)
https://nextstrain.org/groups/neherlab/ncov/belgium?c=gt-S_299,613

**Description**
Examination of the latest neherlab/Europe Nextstrain build showed a distinct lineage with 4 new spike mutations (S:29A, S:250I, S:299I, S:613H) on top of the Delta spike mutations. Further investigation using cov-spectrum showed that the lineage is widespread in Europe, in particular in France, Belgium and the Netherlands and is growing fast in proportion. In many countries examined, the growth advantage seems to be on the order of 10-40%. The proportion of sequences belonging to this lineage seem to reach already 1-10% in some countries. The number of spike mutations is indicative of a monophyletic origin.

The first few sequences on GISAID with these 4 mutations were obtained here:

strain | date | region_original | country_original | division | Ns
-- | -- | -- | -- | -- | --
hCoV-19/Japan/IC-1357/2021 | 2021-06-11 | Asia | Japan |   | 0
hCoV-19/Morocco/RA207/2021 | 2021-06-20 | Africa | Morocco |   | 0
hCoV-19/Israel/SMC-7005096/2021 | 2021-06-28 | Asia | Israel |   | 180
hCoV-19/Netherlands/GR-RIVM-47410/2021 | 2021-06-30 | Europe | Netherlands | Groningen | 0
hCoV-19/Belgium/UZA-UA-47848402/2021 | 2021-07-03 | Europe | Belgium | Antwerpen | 0
hCoV-19/Germany/NI-RKI-I-187479/2021 | 2021-07-04 | Europe | Germany | Niedersachsen | 124
hCoV-19/Northern Ireland/PHEC-M30AM407/2021 | 2021-07-05 | Europe | United Kingdom | Northern Ireland | 0
hCoV-19/Belgium/UZA-UA-CV2253887754/2021 | 2021-07-06 | Europe | Belgium | Antwerpen | 3
hCoV-19/France/PDL-IPP15314/2021 | 2021-07-06 | Europe | France | Pays de La Loire | 475
hCoV-19/France/IDF-HMN-21072080322/2021 | 2021-07-06 | Europe | France | Ile de France | 248
hCoV-19/France/GES-HMN-21072120416/2021 | 2021-07-07 | Europe | France | Grand Est | 392
hCoV-19/France/BFC-HMN-21072220511/2021 | 2021-07-07 | Europe | France | Bourgogne-France-Comté | 254
hCoV-19/Canada/QC-L00365546001/2021 | 2021-07-08 | North America | Canada | Quebec | 571
hCoV-19/ITA/ABR-A39164/2021 | 2021-07-09 | Europe | Italy | Abruzzo | 0

The lineage has been observed in a significant number of countries (Morocco, Belgium, Switzerland, Netherlands).

The second earliest sequences is from Morocco and Morocco does not sequence much at all (only a dozen in the last few months). Together with the fact that the lineage suddenly appeared in multiple European countries that have historically close ties with Morocco (France, Belgium, Netherlands) makes it quite plausible to me that this lineage was exported from Morocco.

Country | Sequence count
-- | --
Belgium | 482
Denmark | 266
France | 255
Germany | 213
United Kingdom | 106
Netherlands | 89
USA | 86
Italy | 60
Switzerland | 49
Spain | 42
Sweden | 28
Austria | 8
Canada | 6
Norway | 3
South Korea | 3
Finland | 2
Ireland | 2
Israel | 2
Japan | 2
Hong Kong | 1
Iceland | 1
Morocco | 1
Portugal | 1
Romania | 1
Singapore | 1

Strains:
[strains.csv](https://github.com/cov-lineages/pango-designation/files/7145143/strains.csv)

Screenshots:
![image](https://user-images.githubusercontent.com/25161793/132885993-4f559b8c-19fe-40ed-ab91-10bda75fa6f4.png)
https://nextstrain.org/groups/neherlab/ncov/europe/2021-09-09?branchLabel=aa&c=gt-S_29,250,299,613&gmin=15&m=div

Growth in share in all European sequences:
![image](https://user-images.githubusercontent.com/25161793/132886674-d28a52a0-3a06-452f-b08c-0e3fe23423ac.png)
https://cov-spectrum.ethz.ch/explore/Europe/AllSamples/AllTimes/variants/json=%7B%22variant%22%3A%7B%22mutations%22%3A[%22S%3A29A%22%2C%22S%3A250I%22%2C%22S%3A299I%22%2C%22S%3A613H%22]%7D%2C%22matchPercentage%22%3A1%7D

And the corresponding data as a table (note in the table the proportion is of Delta whereas in the graph it's proportion of all sequences):

year | week | delta_count (Europe) | new_lineage_count (Europe) | proportion of lineage of all Delta (Europe)
-- | -- | -- | -- | --
2021 | 26 | 9509 | 3 | 0.03%
2021 | 27 | 15930 | 13 | 0.08%
2021 | 28 | 17237 | 24 | 0.14%
2021 | 29 | 22934 | 54 | 0.24%
2021 | 30 | 23008 | 103 | 0.45%
2021 | 31 | 22082 | 195 | 0.88%
2021 | 32 | 28400 | 306 | 1.08%
2021 | 33 | 36941 | 307 | 0.83%
2021 | 34 | 14781 | 225 | 1.52%
2021 | 35 | 5344 | 121 | 2.26%
2021 | 36 | 58 | 1 | 1.72%

This is Usher with the selected strain names
![image](https://user-images.githubusercontent.com/25161793/132891765-05b03279-befd-4f01-8a56-391bfb3996d3.png)

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/singleSubtreeAuspice_genome_5e61a_b90a90.json?branchLabel=Spike%20mutations

**Comment by chaoran-chen** [comment]
This shows the prevalence of the four mutations among delta sequences in different countries.

![prevalence_within_delta](https://user-images.githubusercontent.com/18666552/132906674-189a0833-c3b1-4d39-8d8f-3cadffe445f6.png)

**Comment by c19850727** [comment]
looks like the same one as #206 (proposed new lineage 4) ?

**Comment by FedeGueli** [comment]
Yes it is the same one as lineage 4 proposed in #206 ,
probably it needs a second chance to be designated.

**Comment by rmcolq** [comment]
I'm looking into designating it now

**Comment by corneliusroemer** [comment]
Closed in https://github.com/cov-lineages/pango-designation/commit/4ba575250d207bb528693ba9250f8c35e31739d7

**Comment by corneliusroemer** [comment]
@rmcolq I closed it because you've designated it, but it probably should still get the `designated` label

**Comment by shay671** [comment]
One of the interesting thing is how a variant is spreading in correlation to it's most closest branch. 
In case of AY.33 this is the clade described as clade D (ref in the end of this message), the clade which included N_G215C (among other 12 mutations over the base of Delta and 9 over a base of 4 out of the 5 Delta clades) and which is the one taking over the world. 
AY.33 has 4 mutations over Delta clade D (all in S1 as mentioned). 

I took from GISAID the samples of Delta D from 3 countries in which AY.33 is spreading and also the AY.33 samples. 
Delta D was taken using the mutation signature : Spike_L452R,Spike_P681R,N_G215C 
Ay.33 was taken using the mutations signature : Spike_L452R,Spike_P681R,N_G215C,S_T299I,S_Q613H
![image](https://user-images.githubusercontent.com/80166370/133282357-108fcbe9-e2b4-4947-9ef7-9dd6cd27a1b6.png)

![image](https://user-images.githubusercontent.com/80166370/133282435-66edab98-d8fc-4a98-87fb-6ac81c054e0b.png)

![image](https://user-images.githubusercontent.com/80166370/133282510-8d1cb055-f7aa-46c4-809b-67ab822de052.png)


It is important to mention - In the past i did this analysis of other Delta D based AY variants such as AY.1,AY.3,AY.4 and no such increase was found. 
But this trend is seen in several countries with AY.23 


The paper were Delta D and it's kinetics was shown : 
https://www.medrxiv.org/content/10.1101/2021.08.05.21261642v1

**Comment by shay671** [comment]
The reason for this analysis is as seen with AY.1,AY.3 and AY.4 - this trend may be just the regular increase of Delta D over all the other 4 clades of Delta (meaning that it may be just a reporter for Delta D) which is still occur in many countries.

**Comment by silcn** [comment]
A lineage going quickly up to ~10% and then getting stuck there (Belgium) seems more likely due to lots of introductions in a relatively short period rather than an increase in fitness. For all we know, AY.33 could be dominant in Morocco, and I note that Belgium has a particularly large Moroccan diaspora so there was probably a lot of travel between the two countries over the summer holidays. Germany and the UK appear to show a similar effect - I wouldn't read much into the figures from the latest week where sequencing is clearly far from complete.

Denmark does not have a large North African population so possible that introductions there would be later on average, as they would tend to come from other European countries rather than Africa.

AY.33 still might have an advantage but surely it would have to be small. This reminds me more of Kappa vs Alpha in the UK than Delta vs Alpha.

**Comment by corneliusroemer** [comment]
@rmcolq @chrisruis

I've looked at the data around this lineage AY.33 more closely and it may be worth to designate a parent of it and turn what was AY.33 into AY.33.1 (or redesignated all as AY.34 and AY.34 to avoid problems).

Will submit the strains for the S:29A, S:250I parent lineage in another issue for your consideration?

Maybe hold off making a release until then.

**Comment by corneliusroemer** [comment]
@rmcolq I've now submitted a proposal for the parent lineage as #219 and will thus reopen this issue so that both can be considered together.

**Comment by corneliusroemer** [comment]
Good news, looks like the fast AY.33 expansion was driven by export from Morocco where it grew due to founder effect (where recent sequencing showed it's at ~50%, so not dominant!)
![image](https://user-images.githubusercontent.com/25161793/134777820-0d26c46e-50e7-4b9a-a407-878e060063ba.png)

**Comment by corneliusroemer** [comment]
It's a good thing this lineage has not been released. It should be released only once #219 has been resolved and removed until then.
@rmcolq @chrisruis

**Comment by chrisruis** [comment]
Based on discussion in #219, we've updated AY.33 in [v1.2.79](https://github.com/cov-lineages/pango-designation/releases/tag/v1.2.79) to start on the branch with A21647G (S:T29A), more details in #219
